packages/block-library/src/group/edit.native.js

```diff
@@ -64,6 +68,19 @@ function GroupEdit( {
 					! hasInnerBlocks &&
 					isFullWidth &&
 					styles.fullWidth,
+				mergedStyle,
+				isSelected &&
+					hasInnerBlocks &&
+					mergedStyle?.backgroundColor && {
+						paddingBottom:
+							styles.hasBackgroundAppender.paddingBottom,
+					},
```

Comment:
It could be moved to the scss file

```suggestion
					mergedStyle?.backgroundColor && styles.hasBackgroundAppender,
```